src/server.cpp

```diff
@@ -1718,17 +1718,62 @@ void Server::SendHUDSetParam(session_t peer_id, u16 param, const std::string &va
 	Send(&pkt);
 }
 
-void Server::SendSetSky(session_t peer_id, const video::SColor &bgcolor,
-		const std::string &type, const std::vector<std::string> &params,
-		bool &clouds)
+void Server::SendSetSky(session_t peer_id, const SkyboxParams &params)
 {
 	NetworkPacket pkt(TOCLIENT_SET_SKY, 0, peer_id);
-	pkt << bgcolor << type << (u16) params.size();
 
-	for (const std::string &param : params)
-		pkt << param;
+	// Handle prior clients here
+	if (m_clients.getProtocolVersion(peer_id) < 39) {
+		pkt << params.bgcolor << params.type << (u16) params.textures.size();
+
+		for(size_t i=0; i<params.textures.size(); i++)
```

Comment:
you missed this one, not essential though

`for (const std::string &texture : params.params)`

Reply:
This is stupid, you can fix it by yourself

Reply:
We do not have time to fix contributors' PRs. It is not our responsibility or obligation.
Fixing something ourselves only makes sense if it is certain that it is the very last change needed, and the PR is close to being approved, neither of which is the case here.

This seems resolved, but only sfan5 should resolve this comment.